docs/howto-windows.md

```diff
@@ -0,0 +1,28 @@
+# RLLIB SUMO Docker environment for Windows
+
+
+Step by step configuration and use:
+
+1. install docker for windows
+
+(https://hub.docker.com/editions/community/docker-ce-desktop-windows/)
+
+2. install VcXsrv Windows X Server (https://sourceforge.net/projects/vcxsrv/files/latest/download)
+
+(follow steps from https://dev.to/darksmile92/run-gui-app-in-linux-docker-container-on-windows-host-4kde)
+
+3. build the docker image using the docker cmd bash script
+
+`./docker-cmd-windows.sh -n=marl-parking-devel -b -c -r`
+
+4. use the following command line: to set up VcXsrv:
+
+`set-variable -name DISPLAY -value 192.168.xx.xx:0.0`
+
+(/!\ replace "192.168.xx.xx" with your ip you can obtain using ipconfig command line)
+
+5. use the following command line to run your docker image:
+
+(of course replace "GroupID, UserID and FolderPath" accordingly
```

Comment:
Can this be embedded in docker-cmd-windows.sh with 
* DISPLAY
* UID & GID
as command-line parameters?